actions/nuxeo-delete-document-button.html

```diff
@@ -0,0 +1,122 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../iron-icon/iron-icon.html">
+<link rel="import" href="../../iron-icons/iron-icons.html">
+<link rel="import" href="../../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../paper-tooltip/paper-tooltip.html">
+<link rel="import" href="../../paper-button/paper-button.html">
+<link rel="import" href="../../nuxeo-elements/nuxeo-operation.html">
+<link rel="import" href="../widgets/nuxeo-dialog.html">
+<link rel="import" href="../nuxeo-i18n-behavior.html">
+<link rel="import" href="../nuxeo-filters-behavior.html">
+
+<!--
+A button element for moving document to trash (if available server side) or permanently delete if hard set to true.
+
+Example:
+
+    <nuxeo-delete-document-button document="[[document]]"></nuxeo-delete-document-button>
+
+    <nuxeo-delete-document-button document="[[document]]" hard></nuxeo-delete-document-button>
+
+@group Nuxeo UI
+@element nuxeo-delete-document-button
+-->
+<dom-module id="nuxeo-delete-document-button">
+
+  <template>
+    <style>
+      :host {
+        display: inline-block;
+      }
+    </style>
+
+    <nuxeo-operation id="deleteOp" op="Document.Delete" input="[[document.uid]]" sync-indexing></nuxeo-operation>
```

Comment:
Now that we end up with 2 different operations for `Delete` and `Trash` instead of the `hard` property (passed in headers) maybe would be better to have 2 separate document actions (as a future improvement).